tests/OpenRealEstate.Validation.Tests/MediaValidatorTests.cs

```diff
@@ -41,5 +41,20 @@ public void GivenNoUrl_Validate_ShouldHaveAValidationError()
         {
             _mediaValidator.ShouldHaveValidationErrorFor(media => media.Url, "");
         }
+
+        [Fact]
+        public void GivenAnId_Validate_ShouldNotHaveAValidationError()
+        {
+            const string validId = "a";
+            _mediaValidator.ShouldNotHaveValidationErrorFor(media => media.Id, validId);

```

Comment:
don't need to define a const if it's only going to be used once in the code.

the compiler also is smart enough to "know" this and it make it a const during JIT or whatever.

```suggestion
            _mediaValidator.ShouldNotHaveValidationErrorFor(media => media.Id, "a");

```

Reply:
I know, just trying the code look cleaner